hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaRolloutManagement.java

```diff
@@ -385,13 +386,11 @@ private RolloutGroup fillRolloutGroupWithTargets(final JpaRollout rollout, final
         final List<Long> readyGroups = RolloutHelper.getGroupsByStatusIncludingGroup(rollout.getRolloutGroups(),
                 RolloutGroupStatus.READY, group);
 
-        final long targetsInGroupFilter = DeploymentHelper.runInNewTransaction(txManager,
-                "countAllTargetsByTargetFilterQueryAndNotInRolloutGroups",
-                count -> targetManagement.countByRsqlAndNotInRolloutGroups(readyGroups, groupTargetFilter));
+        final long targetsInGroupFilter = inNewTransaction("countAllTargetsByTargetFilterQueryAndNotInRolloutGroups",
+                count -> targetManagement.countByRsqlAndNotInRolloutGroups(readyGroups, groupTargetFilter)).orElse(0L);
         final long expectedInGroup = Math.round(group.getTargetPercentage() / 100 * (double) targetsInGroupFilter);
-        final long currentlyInGroup = DeploymentHelper.runInNewTransaction(txManager,
-                "countRolloutTargetGroupByRolloutGroup",
-                count -> rolloutTargetGroupRepository.countByRolloutGroup(group));
+        final long currentlyInGroup = inNewTransaction("countRolloutTargetGroupByRolloutGroup",
+                count -> rolloutTargetGroupRepository.countByRolloutGroup(group)).orElse(0L);
```

Comment:
maybe we don't need to have an optional with a 'default' value, maybe we can delegate the exception to the catch block?